source/index.ts

```diff
@@ -168,11 +171,14 @@ export default class Pageres extends EventEmitter {
 				return this.viewport({url: source.url, sizes, keywords}, options);
 			}
 
-			for (const size of sizes) {
-				this.sizes.push(size);
-				// TODO: Make this concurrent
-				this.items.push(await this.create(source.url, size, options));
-			}
+			this.items.push(...await pMap(
```

Comment:
This is not very readable. Make it a variable and then in a new statement, you can use destructuring instead of "push".